tests/unit/amo/components/TestAddonHead.js

```diff
@@ -197,4 +207,73 @@ describe(__filename, () => {
 
     expect(root.find('meta[name="last-modified"]')).toHaveLength(0);
   });
+
+  it.each([CLIENT_APP_ANDROID, CLIENT_APP_FIREFOX])(
+    'renders alternate links with hreflang for %s',
+    (clientApp) => {
+      const addon = createInternalAddon(fakeAddon);
+      const baseURL = 'https://example.org';
+
+      const _hrefLangs = ['fr', 'en-US'];
+      const _config = getFakeConfig({ baseURL });
+      const { store } = dispatchClientMetadata({ clientApp });
+
+      const root = render({ _config, _hrefLangs, addon, store });
+
+      expect(root.find('link[rel="alternate"]')).toHaveLength(
+        _hrefLangs.length,
+      );
+      _hrefLangs.forEach((locale, index) => {
+        expect(root.find('link[rel="alternate"]').at(index)).toHaveProp(
+          'hrefLang',
+          locale,
+        );
+        expect(root.find('link[rel="alternate"]').at(index)).toHaveProp(
+          'href',
+          `${baseURL}/${locale}/${clientApp}/addon/${addon.slug}/`,
+        );
+      });
+    },
+  );
+
+  it('renders alternate links for aliased locales', () => {
+    const addon = createInternalAddon(fakeAddon);
+    const baseURL = 'https://example.org';
+    const clientApp = CLIENT_APP_FIREFOX;
+
+    const _hrefLangs = ['x-default'];
+    const hrefLangsMap = {
+      'x-default': 'en-US',
+    };
+    const _config = getFakeConfig({ baseURL, hrefLangsMap });
+    const { store } = dispatchClientMetadata({ clientApp });
+
+    const root = render({ _config, _hrefLangs, addon, store });
+
+    expect(root.find('link[rel="alternate"]')).toHaveLength(_hrefLangs.length);
+    expect(root.find('link[rel="alternate"]').at(0)).toHaveProp(
+      'hrefLang',
+      _hrefLangs[0],
```

Comment:
Nit: I wonder if the test might be a bit more readable if you were to set `const`s equal to `x-default` and `en-US` and then use those to set values for `_hrefLangs` and `hrefLangsMap`, and then in the asserts you could just use the `const`s again, rather than things like `_hrefLangs[0]` and `hrefLangsMap[_hrefLangs[0]]`?

Reply:
yep